tests/Microsoft.Identity.Web.Test.Integration/AcquireTokenForAppIntegrationTests.cs

```diff
@@ -114,9 +115,38 @@ public void ApplicationOptionsIncludeClientSecret(string clientSecret)
             }
         }
 
+        [Theory]
+        [InlineData(null, false)]
+        [InlineData("", false)]
+        [InlineData("notAUri", false)]
+        [InlineData("htt://nonhttp/", false)]
+        [InlineData("https://login.microsoftonline.com/", true)]
+        [InlineData("https://login.microsoftonline.com", true)]
+        [InlineData("https://cats.b2clogin.com/", true)]
+        [InlineData("https://cats.b2clogin.com/signout-callback-oidc", true)]
+        [InlineData("http://cats.b2clogin.com/signout-callback-oidc", true)]
+        public void ValidateRedirectUriFromMicrosoftIdentityOptions(
+            string redirectUri,
+            bool isTrue)
```

Comment:
I think you can come up with a better name here :)

Reply:
expectConfiguredUri?

Reply:
expected?

Reply:
lol...yes true...this was a quick thing for the test. 

---
In reply to: [420322679](https://github.com/AzureAD/microsoft-identity-web/pull/144#discussion_r420322679) [](ancestors = 420322679)